docs/diagrams/UiClassDiagram.puml

```diff
@@ -33,21 +37,31 @@ UiManager -down-> "1" MainWindow
 MainWindow *-down->  "1" CommandBox
 MainWindow *-down-> "1" ResultDisplay
 MainWindow *-down-> "1" BuyerListPanel
+MainWindow *-down-> "1" MeetUpListPanel
+MainWindow *-down-> "1" PropertyListPanel
 MainWindow *-down-> "1" StatusBarFooter
 MainWindow --> "0..1" HelpWindow
 
 BuyerListPanel -down-> "*" BuyerCard
+MeetUpListPanel -down-> "*" MeetUpCard
+PropertyListPanel -down-> "*" PropertyCard
 
 MainWindow -left-|> UiPart
 
 ResultDisplay --|> UiPart
 CommandBox --|> UiPart
 BuyerListPanel --|> UiPart
+MeetUpListPanel --|> UiPart
+PropertyListPanel --|> UiPart
 BuyerCard --|> UiPart
+MeetUpCard --|> UiPart
+PropertyCard --|> UiPart
 StatusBarFooter --|> UiPart
 HelpWindow --|> UiPart
 
 BuyerCard ..> Model
+MeetUpCard ..> Model
+PropertyCard ..> Model
```

Comment:
You might need to export the puml file as png and place it in the images folder, else the diagram might not actually get updated in the DG.